server/server.go

```diff
@@ -140,13 +140,7 @@ func (s *Server) loadContainer(id string) error {
 		return err
 	}
 
-	var img *pb.ImageSpec
-	image, ok := m.Annotations[annotations.Image]
-	if ok {
-		img = &pb.ImageSpec{
-			Image: image,
-		}
-	}
+	img, _ := m.Annotations[annotations.Image]
```

Comment:
This doesn't quite look correct... Maybe check `ok` first to see if we have an Image annotation, and use "" if we don't?